src/System.Numerics.Vectors/ref/Configurations.props

```diff
@@ -9,6 +9,7 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      uapaot;
```

Comment:
This is wrong, why do we want a new configuration if uap and uapaot produce the exact same dll?

Reply:
Yes, you are right.

Reply:
Deleted in https://github.com/dotnet/corefx/pull/27663/